docs/how_to/documenting_new_feature_toggles.rst

```diff
@@ -0,0 +1,74 @@
+.. _documenting_new_feature_toggles:
+
+=======================================
+How to: Documenting new feature toggles
+=======================================
+
+Feature toggles are boolean values that enable or disable the use of a certain feature. There are many of those across the Open edX codebase: in the form of Django settings, Configuration models, Waffle switches and flags, for example. It this important that these feature toggles be correctly annotated to address the following scenarios:
+
+- Generation of human-readable documentation of existing feature toggles for Open edX operators: instead of grepping a large codebase, operators (sysadmins and product owners, for instance) should be able to search an online document to learn how to use and enable features on their platforms.
+- Automated discovery and verification of feature toggles on a running platform: this allows, for instance, the comparison of feature toggles between different environments at runtime.
+
+Documentation format
+====================
+
+The requirements for feature toggle annotations in the Open edX codebase are extensively documented in `OEP-17 <https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html>`__.
+
+The documentation format used to annotate feature toggles is stored in the code-annotations repository: `feature_toggle_annotations.yaml <https://github.com/edx/code-annotations/blob/master/code_annotations/config_and_tools/config/feature_toggle_annotations.yaml>`__.
+
+Annotation fields
+=================
+
+Most annotation fields are self-explanatory. In this section we describe in more details some of the more complex annotation fields.
```

Comment:
I might include a full example early on (using `#`).  You could also note that it could be included in a docstring, for those that don't know how code-annotations work.

Reply:
:ok_hand: